### ISSUE TITLE
ApolloError: Unexpected token ﻿ in JSON at position 0

### ISSUE THREAD

**Issue by KhadijaMahanga** [WebDevStudios/nextjs-wordpress-starter#561]
I am getting this error. Can anyone assist. However I can see the data is being returned and statusCode 200.
 
```
ApolloError: Unexpected token ﻿ in JSON at position 0
    at new ApolloError (/../node_modules/@apollo/client/errors/errors.cjs.js:31:28)
    at /.../node_modules/@apollo/client/core/core.cjs.js:1493:19
    at both (/.../node_modules/@apollo/client/utilities/utilities.cjs.js:963:53)
    at /..../node_modules/@apollo/client/utilities/utilities.cjs.js:956:72
    at new Promise (<anonymous>)
    at Object.then (/.../node_modules/@apollo/client/utilities/utilities.cjs.js:956:24)
    at Object.error (/.../node_modules/@apollo/client/utilities/utilities.cjs.js:964:49)
    at notifySubscription (/.../node_modules/zen-observable/lib/Observable.js:140:18)
    at onNotify (/..../node_modules/zen-observable/lib/Observable.js:179:3)
    at SubscriptionObserver.error (/..../node_modules/zen-observable/lib/Observable.js:240:7) {
  graphQLErrors: [],
  networkError: SyntaxError [ServerParseError]: Unexpected token ﻿ in JSON at position 0
      at JSON.parse (<anonymous>)
      at /..../node_modules/@apollo/client/link/http/http.cjs.js:18:25
      at processTicksAndRejections (internal/process/task_queues.js:93:5) {
    response: Response {
      size: 0,
      timeout: 0,
      [Symbol(Body internals)]: [Object],
      [Symbol(Response internals)]: [Object]
    },
    statusCode: 200,
    bodyText: '﻿{"data":{"homepageSettings":{"frontPage":{"seo":{"breadcrumbs":[{"text":"Home","url":"\\/","__typename":"SEOPostTypeBreadcrumbs"}],"canonical":...
```

**Comment by docforcedev** [comment]
My issue was this: https://github.com/WebDevStudios/nextjs-wordpress-starter/blob/main/lib/wordpress/connector.js#L30

My endpoint isn't `/grahpql` but is `/gql` due to custom redirects we have setup for `/graph*` in wordpress.

**Comment by KhadijaMahanga** [comment]
@docforcedev where you getting the exact same error? My case here the token is an invisible character...the response seems to hace a 200 Status Code and bodyText has data that's intact

**Comment by docforcedev** [comment]
@KhadijaMahanga Can you post the entire output? It's cut off at the end

**Comment by ravewebdev** [comment]
@KhadijaMahanga As @docforcedev requested, can you please paste the full, non-truncated output? Our team hasn't encountered this specific issue so we'll need more info to go on. Is there anything special about your setup (WordPress or frontend)?

**Comment by KhadijaMahanga** [comment]
@ravewebdev I finally realized the issue was coming from wordpress setup. So I am closing this issue. I pasted the `bodyText` on hexEditor..and there were actual invisible characters  that could not be parse to JSON. I have restarted my wp setup and all good now.
![screenshot_from_2021-07-16_10-23-41](https://user-images.githubusercontent.com/7962097/126591943-58e4baab-2ce6-4e5f-bbae-3dc1c406092f.png)
